distsql/select_result.go

```diff
@@ -111,6 +112,9 @@ func (r *selectResult) fetch(ctx context.Context) {
 		} else {
 			result.result = resultSubset
 			r.memConsume(int64(resultSubset.MemSize()))
+			if r.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl != nil {
```

Comment:
When we call `distsql.go:Analyze`, the `r.ctx` is not initialized, which would cause panic.